### PR TITLE
Kbase migration - User perms

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Migrate https://kb.timescale.cloud/en/articles/2752453-inviting-team-members-and-managing-privileges to docs

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescale-cloud/user-management.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    shows in the `Pending invitations` list. You can click `Withdraw invitation`
+    to remove an invitation before it has been accepted.
+1.  When they accept the invitation, the user details show in the `Members`
+    list. You can edit a member role by selecting a new role in the list. You
+    can delete a member by clicking the delete icon in the list.
+
+## Service users
+You can create user accounts for accessing your services.
+
+<highlight="important">
```
Comment (Loquacity) [inline timescale-cloud/user-management.md:35]:
```suggestion
<highlight type="important">
```

File: timescale-cloud/user-management.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# User Management
+You can add new users, and manage existing users, in the Managed Service for
+TimescaleDB (MST) console. New users can be added to an entire project, or a
```
Comment (ryanbooz) [inline timescale-cloud/user-management.md:3]:
```suggestion
You can add new users, and manage existing users, in the Timescale Cloud console. New users can be added to an entire project, or a
```

File: timescale-cloud/user-management.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+|Role|Invite more users|Modify billing information|Manage services|Start and stop services|View service information|
+|-|-|-|-|-|-|
+|Admin|✅|✅|✅|✅|✅|
+|Operator|❌|❌|✅|✅|✅|
+|Developer|✅|❌|✅|❌|✅|
+|Read-only|❌|❌|❌|❌|✅|
+
+### Procedure: Adding project members
+1.  [Log in to your MST account][mst-login]. By default, you start in the
```
Comment (ryanbooz) [inline timescale-cloud/user-management.md:18]:
```suggestion
1.  [Log in to your Timescale Cloud account][mst-login]. By default, you start in the
```

File: timescale-cloud/user-management.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Service users
+You can create user accounts for accessing your services.
+
+<highlight type="important">
+Your service must be running before you can manage users.
+</highlight>
+
+### Procedure: Adding service users
+1.  [Log in to your MST account][mst-login]. By default, you start in the
```
Comment (ryanbooz) [inline timescale-cloud/user-management.md:40]:
```suggestion
1.  [Log in to your Timescale Cloud account][mst-login]. By default, you start in the
```

**Review by ryanbooz — CHANGES_REQUESTED** [review]

**Comment by Loquacity** [comment]
image

**Review by ryanbooz — APPROVED** [review]

**Pull request merged**